docs/getting_started_guide/Installation_Guide/Advanced_Configuration.rst

```diff
@@ -73,7 +73,7 @@ that variables beginning with 'cli::' are only used internally by
 | cli::network::netmask          | Netmask of the system                     |
 +--------------------------------+-------------------------------------------+
 | cli::network::set_up_nic       | Whether to set up the network interface;  |
-|                                | *true* or *false*                         |
+|                                | *yes* or *no*                             |
```

Comment:
simp config asks yes or no but prints out true or false in the persisted files.  So, which do you want to list here?  Also, whichever convention you choose, would you please make sure it is used consistently in the entire table?